solr/core/src/java/org/apache/solr/servlet/HttpSolrCall.java

```diff
@@ -736,107 +709,21 @@ void destroy() {
     }
   }
 
-  // TODO using Http2Client
-  private void remoteQuery(String coreUrl, HttpServletResponse resp) throws IOException {
-    HttpRequestBase method;
-    HttpEntity httpEntity = null;
+  protected void sendRemoteQuery() throws IOException {
+    SolrRequestInfo.setRequestInfo(new SolrRequestInfo(req, new SolrQueryResponse(), action));
+    mustClearSolrRequestInfo = true;
+
+    HttpClient httpClient = cores.getDefaultHttpSolrClient().getHttpClient();
```

Comment:
FYI using the underlying Jetty `HttpClient`, one layer deeper than the `Http2SolrClient`, thus skips over the authentication plugin interception stuff since it's not registered at quite that depth.  I abandoned a previous PR that attempted to lower that logic down here.